biggus/__init__.py

```diff
@@ -860,6 +914,7 @@ def __init__(self, array, broadcast, leading_shape=()):
         shape = self._shape_from_broadcast_dict(self.array.shape, broadcast)
         for length in leading_shape:
             if length < 1:
+                print leading_shape
```

Comment:
Actually I think that is a leftover "debugging" print, right?  I'd rather if length was printed in the `ValueError` below.